.npmignore

```diff
@@ -1 +0,0 @@
-src
```

Comment:
Now using `files` array in `packge.json` to say what should be installed when library is installed in a project, instead of ignoring what shouldn't